### PR TITLE
feat(Gemini): added option to show title

### PR DESCRIPTION
**Pull request by tschogge**
## Description 
feat(gemini): added option to show title
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
When show title is activated

image

When show title is deactivated

image

</details>

### CODE REVIEW

File: package-lock.json
Comment (theusaf) [inline package-lock.json:1]: Please revert changes to this file

File: websites/G/Gemini/presence.ts [hunk 1]
```diff
@@ -13,10 +13,27 @@ presence.on("UpdateData", async () => {
...
+					).textContent ?? "Thinking of a new Prompt...";
+
+				// Show word count
+				const elem = document.querySelectorAll(
+						"p.query-text-line,div.response-optimization.markdown"
+					),
+					// eslint-disable-next-line no-one-time-vars/no-one-time-vars
+					askedWOrds = elem[0].textContent.split(" ").length,
+					// eslint-disable-next-line no-one-time-vars/no-one-time-vars
+					answerWords = elem[1].textContent.split(" ").length;
```
Comment (theusaf) [inline websites/G/Gemini/presence.ts:34]: Unless required, do not disable eslint rules

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/G/Gemini/presence.ts (hunk 1 above)
Comment (tschogge) [reply websites/G/Gemini/presence.ts]: okay resolved it

File: package-lock.json
Comment (tschogge) [reply package-lock.json:1]: reverted

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**